tests/network/vmi_multus.go

```diff
@@ -679,6 +679,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 		getSriovVmi := func(networks []string, cloudInitNetworkData string) *v1.VirtualMachineInstance {
 
 			withVmiOptions := []libvmi.Option{
+				libvmi.WithCloudInitNoCloudUserData(tests.GetFedoraToolsGuestAgentUserData(), false),
```

Comment:
Please consider the comment of 
https://github.com/kubevirt/kubevirtci/pull/648#discussion_r676103464